core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -128,6 +129,15 @@ case class GroupSummary(state: String,
                         members: List[MemberSummary])
 
 /**
+ * Case class used to represent group metadata (including the group coordinator) for the DescribeGroup API
+ */
+case class ConsumerGroupSummary(state: String,
+                                protocolType: String,
```

Comment:
Thanks. With the first suggestion, there already is a `describeConsumerGroup` method. Unless you prefer this suggestion (using a different method name) I'll go ahead with your second suggestion.